pkg/apis/v1alpha1/vpc_types.go

```diff
@@ -0,0 +1,96 @@
+/* Copyright � 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+   package v1alpha1
+
+   import (
+	   corev1 "k8s.io/api/core/v1"
+	   metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+   )
+   
+   type VPCStatusCondition string
+   
+   const (
+	   VPCReady    VPCStatusCondition = "Ready"
+	   VPCNotReady VPCStatusCondition = "NotReady"
+   )
+   
+   //+kubebuilder:object:root=true
+   //+kubebuilder:subresource:status
+   //+kubebuilder:storageversion
+   
+   // VPC is the Schema for the VPC API
+   type VPC struct {
+	   metav1.TypeMeta   `json:",inline"`
+	   metav1.ObjectMeta `json:"metadata,omitempty"`
+   
+	   Spec   VPCSpec   `json:"spec,omitempty"`
+	   Status VPCStatus `json:"status,omitempty"`
+   }
+   
+   //+kubebuilder:object:root=true
+   
+   // VPCList contains a list of VPC
+   type VPCList struct {
+	   metav1.TypeMeta `json:",inline"`
+	   metav1.ListMeta `json:"metadata,omitempty"`
+	   Items           []VPC `json:"items"`
+   }
+   
+   // VPCSpec defines VPC configuration
+   type VPCSpec struct {
+	   // DHCP config specified at VPC level is applicable to all the Subnets under the VPC.
+	   // +optional
+	   DHCPConfig DHCPConfig `json:"dhcpConfig"`
```

Comment:
Maybe we can remove this field for now, before we are clear of what DHCP and DNS config to support in phase 1. @lxiaopei

Reply:
OK.Sure, We can remove it for now.